From 38f1e6229fac06f00a20f54b80875e029f208c95 Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
Date: Wed, 29 Sep 2021 22:19:32 +0300
Subject: [PATCH 145/322] [FIO extras] mx6ullevk: drop MODULE_FUSE and
 OF_SYSTEM_SETUP for SPL

MODULE_FUSE adds ~8K of binary size to SPL where it's only used for 1
check to see if USB has been fused off.

For FIO use-case we only enable USB for mfgtool, where USB fusing
would not apply.

Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 include/configs/mx6ullevk.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/configs/mx6ullevk.h b/include/configs/mx6ullevk.h
index bc70966c1c2..f2da9d95bc3 100644
--- a/include/configs/mx6ullevk.h
+++ b/include/configs/mx6ullevk.h
@@ -294,6 +294,9 @@
 #define CONFIG_VIDEO_BMP_RLE8
 #define CONFIG_VIDEO_BMP_LOGO
 #endif
+#else /* for SPL_BUILD disable these subsystems */
+#undef CONFIG_MODULE_FUSE
+#undef CONFIG_OF_SYSTEM_SETUP
 #endif
 
 #endif
-- 
2.34.1

